eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -348,7 +348,7 @@ jobs:
 
       - task: Maven@3
         displayName: 'Generate aggregate code coverage report'
-        condition: and(eq(variables['OsName'], 'Windows'), eq(variables['JavaTestVersion'], '1.11'), eq('${{ parameters.SDKType }}', 'client'))
+        condition: and(eq(variables['Agent.OS'], 'Windows'), eq(variables['JavaTestVersion'], '1.11'), eq('${{ parameters.SDKType }}', 'client'))
```

Comment:
The same change needs to be made on [line number 413](https://github.com/Azure/azure-sdk-for-java/pull/18771/files#diff-78f6357ab2cf93b8a11c1fb81a2ba78da1e364b1d14be096d9076e0ccfb2b324R413) to publish the coverage results.

Reply:
Also, looks like `Agent.OS` resolves to `Windows_NT` - https://dev.azure.com/azure-sdk/public/_build/results?buildId=699373&view=logs&j=a1083672-64bd-548a-2539-25550518eef9&t=9bb0c22f-7328-5f8d-38b1-bff61c14501e

Reply:
Fixed. Thanks.